### PR TITLE
Update alert-policies.md

### LINKED ISSUE #11929: Pointing out that default alerts are configured to have mail sent to tenantadmins group
I wonder if there is an opportunity to point out in tis article that default alerts are automatically sent to Tenantadmins and who that implies (Global Admins).

Have had feedback from customers that this isn't really clear and has caused some confusion. Many  global admin roles do not have email and default alerts can be missed.

Is this documented anywhere else if not in this article?

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 9f2879ba-e58e-cd47-f488-10d153c31418
* Version Independent ID: 391a2c05-d63d-e84e-7c7d-80c3b3518419
* Content: [Microsoft 365 alert policies - Microsoft Purview (compliance)](https://learn.microsoft.com/en-us/microsoft-365/compliance/alert-policies?view=o365-worldwide#default-alert-policies)
* Content Source: [microsoft-365/compliance/alert-policies.md](https://github.com/MicrosoftDocs/microsoft-365-docs/blob/public/microsoft-365/compliance/alert-policies.md)
* Service: **o365-seccomp**
* GitHub Login: @robmazz
* Microsoft Alias: **robmazz**

### PR DESCRIPTION
**Pull request by msbemba**
Added a statement that alerts are sent to tenant admins by default.

Fixes https://github.com/MicrosoftDocs/microsoft-365-docs/issues/11929

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/compliance/alert-policies.md [hunk 1]
```diff
@@ -119,7 +119,7 @@ You can also define user tags as a condition of an alert policy. This results in
 
 ## Default alert policies
 
-Microsoft provides built-in alert policies that help identify Exchange admin permissions abuse, malware activity, potential external and internal threats, and information governance risks. On the **Alert policies** page, the names of these built-in policies are in bold and the policy type is defined as **System**. These policies are turned on by default. You can turn off these policies (or back on again), set up a list of recipients to send email notifications to, and set a daily notification limit. The other settings for these policies can't be edited.
+Microsoft provides built-in alert policies that help identify Exchange admin permissions abuse, malware activity, potential external and internal threats, and information governance risks. On the **Alert policies** page, the names of these built-in policies are in bold and the policy type is defined as **System**. These policies are turned on by default and sent to Tenant admins. You can turn off these policies (or back on again), set up a list of recipients to send email notifications to, and set a daily notification limit. The other settings for these policies can't be edited.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/alert-policies.md]:
```suggestion
Microsoft provides built-in alert policies that help identify Exchange admin permissions abuse, malware activity, potential external and internal threats, and information governance risks. On the **Alert policies** page, the names of these built-in policies are in bold and the policy type is defined as **System**. These policies are turned on by default and sent to the organization's admins. You can turn off these policies (or back on again), set up a list of recipients to send email notifications to, and set a daily notification limit. The other settings for these policies can't be edited.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@msbemba

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@robmazz Please merge this PR. Thanks

**Comment by yogkumgit** [comment]
Closing as per the comment in #11929

**Pull request closed without merging**